open-api/lang-tools/src/main/java/io/deephaven/lang/completion/results/CompleteInvocation.java

```diff
@@ -63,19 +65,25 @@ public void doCompletion(Set<CompletionItem> results, CompletionRequest request,
             range = replaceToken(replacing, request);
         }
         final String displayCompletion;
-        if (method.getDeclaringClass().getSimpleName().endsWith("Primitives") && method.getDeclaringClass().getPackage().equals(
-            BytePrimitives.class.getPackage())) {
+        if (method.getDeclaringClass().getSimpleName().endsWith("Primitives") &&
+                BytePrimitives.class.getPackage().equals(method.getDeclaringClass().getPackage())) {
             // reduce massive duplication from same-named primitives methods.
             // In the future, when we have better column/type inference, we should be able to delete this workaround
             displayCompletion = "*Primitives." + method.getName() + "(";
         } else {
             displayCompletion = method.getDeclaringClass().getSimpleName() + "." + method.getName() + "(";
         }
         res.append(method.getName()).append("(");
-        CompletionItem result = new CompletionItem(start, length, res.toString(),
-            // let the user know where this method is coming from (include class name in display completion);
-            displayCompletion, range
-        );
+        CompletionItem.Builder result =
+                CompletionItem.newBuilder()
+                .setStart(start)
+                .setLength(length)
+                // let the user know where this method is coming from (include class name in display completion);
+                .setLabel(displayCompletion)
+                .setTextEdit(TextEdit.newBuilder()
+                        .setText(res.toString())
+                        .setRange(range)
+                        .build());
```

Comment:
build() is optional in many cases like this - setters and add- methods for repeated fields are overloaded to take either the builder or the built instance.

It doesn't save on objects created or work to do, but it can help readability? I'm not convinced, but thought i'd point it out for an inlined builder like this.

Reply:
ok, reviewed em, and tried to stick to pure builders.... will look at github diff again for any .build() that don't belong.